src/ripple/test/impl/JSONRPCClient.cpp

```diff
@@ -116,6 +116,11 @@ class JSONRPCClient : public AbstractClient
         {
             Json::Value jr;
             jr["method"] = cmd;
+#if 1  // force use of 2.0 API
+            jr["jsonrpc"] = "2.0";
+            jr["ripplerpc"] = "2.0";
+            jr["id"] = 5;
```

Comment:
The 5 is just a random id that the client should set.  rippled should transfer that id (whatever it is) from the request to the response.

I need to consult with Vinnie about leaving in the #if or not.  He requested that the V2 be tested this way, but I'm not yet sure if we want to abandon testing V1 at this time.